src/main/java/io/airlift/compress/gzip/JdkGzipCodec.java

```diff
@@ -159,7 +159,6 @@ public void reset()
         @Override
         public void end()
         {
-            throw new UnsupportedOperationException("Gzip block compressor is not supported");
```

Comment:
Why is it safe to ignore?

maybe add a code comment or commit message

Reply:
Good feedback, updated the description